updateCertificationStatusApp.sh

```diff
@@ -5,4 +5,20 @@
 # 15 5 * * * cd /tempApps && ./updateCertificationStatusApp.sh
 # This will run it at 0515 UTC, which (depending on DST) is 0015 EST
 
-java -jar certStatusApp-jar-with-dependencies.jar
\ No newline at end of file
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.certStatusApp.$TIMESTAMP.txt
+
+# deal with spaces in filenames by saving off the default file separator (including spaces)
+# and using a different one for this application
```

Comment:
This was borrowed from the other shell files just to be safe. I think the application would save the log file just fine without this.